aunt-leahs-app/src/components/header/SubText.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../../styles.css'
+
+const SubText =(props)=>{
+    return(
+        <div><h3 className="header">{props.title}</h3></div>
```

Comment:
Similar to another comment, I think this should be a div instead of a h3, then any styling should come from the class. Else, h3 styling needs to be specified

Reply:
noted